_docs/README.md

```diff
@@ -60,7 +62,7 @@ If you would like to have Spotify support you will need to setup a Spotify devel
 
 For Spotify auth you will need to add a redirect URI. This is currently set to what we currently use in Grüvee. In your newly created Spotify Dev App, head to the `Edit Settings` section and add `gruvee://spotify_auth` to the Redirect URIs section.
 
-### `FIREBASE_DEV_URI`
+### `FIREBASE_DEV_URI && FIREBASE_PROD_URI`
```

Comment:
I'm a bit hesitant about adding this as part of the steps for setup as we technically only need to use the dev env for things. We also don't give much explanation about the difference between Dev & Prod in these scenarios. I would say we probably don't need this

Reply:
Ahh yes, I thought about it too. But I ran into error while starting the project, that it could not find these variables from `.env` file. Is there a way so that it wouldn't be necessary to provide these PROD variables while developing?

Reply:
Okay I think I see the issue is either:

1. We actually don't set the `ENVIRONMENT` variable to `DEV` in the instructions
2. We import the production variable, but if we don't have it it crashes

I think it's the second issue here so it actually DOES make sense to include this property, but we just don't set it to anything. I don't love this, but for now it shall do I suppose!

Reply:
Yeah, not ideal.. I will take a look at it.

Reply:
Hey. I took a quick look on this issue and seems like the package we are using `react-native-dotenv` has had some issues with undefined variables.

Here is the relevant issues and conversations:

https://github.com/zetachang/react-native-dotenv/issues/38
https://github.com/zetachang/react-native-dotenv/issues/41
https://github.com/zetachang/react-native-dotenv/pull/75

An option as you say is to just add the "missing" variables and don't set them to anything, which is not ideal.
An option as suggested by other people having the same issue, is to switch to different library which does support undefined variables without throwing. As we don't use it in a lot of places it wouldn't take huge effort to change the library.

https://github.com/tusbar/babel-plugin-dotenv-import
https://github.com/luggit/react-native-config

I am leaning towards the first option `babel-plugin-dotenv-import`. Seems more simple to use while still being quite flexible and fixing the issue we are having.

Thoughts about this?

Reply:
Thanks for the research here! I am going to create an issue for us to implement it (I can assign it to you) if you are interested in implementing it! I think it should be relatively straightforward. Either way for now keep the "prod" variables will do for now! 

Thanks

Reply:
#145 
:)